tests/rules/flowbits/test.rules

```diff
@@ -0,0 +1,5 @@
+alert ip any any -> any any (msg:"isset rule need an option"; flowbits:isset; sid:1;)
```

Comment:
As seen in the flowbits documentation, this keyword needs two values, so all the rules written here will result in a parse error due to the lack of the second field.

(cf https://docs.suricata.io/en/latest/rules/flow-keywords.html#flowbits)

Tip: when an SV test fails with `FAILED: got exit code 1, expected 0` this indicates that Suricata finished with an error state, so to debug such cases you should check the `output` directory in the test folder and see what's in `stderr`.

Reply:
I'm not sure what to pass as the second value? I have changed the code for DETECT-FLOWBITS case so that the name is being stored in `ored_variables` array, should I pass this array as the second value?

Reply:
The second value is a variable name, and as such it's not predefined. Just like a code variable, it is user-defined. Does this clear up?